cli/src/helpers/logger.rs

```diff
@@ -55,12 +55,12 @@ pub fn initialize_logger<P: AsRef<Path>>(
     // Filter out undesirable logs. (unfortunately EnvFilter cannot be cloned)
     let [filter, filter2] = std::array::from_fn(|_| {
         let filter = EnvFilter::from_default_env()
-            .add_directive("mio=off".parse().unwrap())
-            .add_directive("tokio_util=off".parse().unwrap())
-            .add_directive("hyper=off".parse().unwrap())
-            .add_directive("reqwest=off".parse().unwrap())
-            .add_directive("want=off".parse().unwrap())
-            .add_directive("h2=off".parse().unwrap());
+            .add_directive("mio=warn".parse().unwrap())
```

Comment:
Would it make sense to only turn these into warns for higher verbosity levels? Is that possible?

Reply:
I set it to off for anything lower than 4, and then warn for 4 and above.